lib/oban/plugins/cron.ex

```diff
@@ -29,6 +29,21 @@ defmodule Oban.Plugins.Cron do
 
   [tzdata]: https://hexdocs.pm/tzdata
   [perjob]: oban.html#module-periodic-jobs
+
+  ## Instrumenting with Telemetry
+  Oban emits the following telemetry event whenever the Cron plugin executes:
+
+  * `[:oban, :plugin, :cron, :start]` — at the point that the `Cron` plugin begins evaluating what cron jobs to enqueue
```

Comment:
After adding instrumentation for four plugins I arrived at the conclusion that I really preferred a single plugin event that was differentiated by metadata. In this case there would be a single `[:oban, :plugin, :start]` event that included a `:plugin` in the metadata.

The Pro telemetry already uses this pattern (https://hexdocs.pm/oban/lifeline.html#instrumenting-with-telemetry). What do you think?

Reply:
That sounds good to me!

Just so that I am consistent, is the `:plugin` value the module name (i.e `__MODULE__`) or is it a downcased atom `:cron`?